src/person/person.route.spec.ts

```diff
@@ -4,54 +4,54 @@ process.env.PORT = '8080';
 import * as chai from 'chai';
 import * as sinon from 'sinon';
 import * as server from '../server';
-import * as userRouter from './user.route';
-import { User } from './user.controller';
-import { UserModel } from './user.model';
+import * as personRouter from './person.route';
+import { Person } from './person.controller';
+import { PersonModel } from './person.model';
 import { OrganizationGroupModel } from '../group/organizationGroup/organizationGroup.model';
-import { IUser } from './user.interface';
+import { IPerson } from './person.interface';
 import { expectError } from '../helpers/spec.helper';
 import { ObjectId } from 'mongodb';
 
 const should = chai.should();
 chai.use(require('chai-http'));
 const expect = chai.expect;
 
-const userExamples: IUser[] = [  
-  <IUser>{
+const personExamples: IPerson[] = [  
+  <IPerson>{
     identityCard: '234567891',
     personalNumber: '3456712',
-    primaryUser: 'mazaltov@surprise.sod',
+    primaryPerson: 'mazaltov@surprise.sod',
```

Comment:
Same here, the name should be changed across all tests

Reply:
Done